Consul.AspNetCore/AgentServiceRegistrationHostedService.cs

```diff
@@ -0,0 +1,30 @@
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Hosting;
+
+namespace Consul.AspNetCore
+{
+	public class AgentServiceRegistrationHostedService : IHostedService
+	{
+		private readonly IConsulClient consulClient;
+		private readonly AgentServiceRegistration registration;
```

Comment:
The naming convention is '_lowerCamelCase' for private fields in the library. Can you please start the variable names with underscore?